src/components/LanguageSelection/LanguageSelection.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+
+import style from './LanguageSelection.scss'
+
+class LanguageSelection extends Component {
```

Comment:
This can be a `PureComponent` I believe, the HOC will update `language`, triggering the render.